packages/dx-react-grid/README.md

```diff
@@ -1,41 +1,57 @@
 # DevExtreme React Grid
 
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [Plugin Overview](plugin-overview)
```

Comment:
These links will be broken once the page is published on npm. We should use absolute URLs for the root readme files.